open-hackathon-server/src/hackathon/__init__.py

```diff
@@ -203,11 +203,12 @@ def init_db():
 
 
 def init_expr_components():
-    from expr import ExprManager, AzureVMExprStarter, AzureHostedDockerStarter, AlaudaDockerStarter
+    from expr import ExprManager, AzureVMExprStarter, AzureHostedDockerStarter, AlaudaDockerStarter, K8sVMExprStarter
     factory.provide("expr_manager", ExprManager)
     factory.provide("alauda_docker", AlaudaDockerStarter)
     factory.provide("azure_docker", AzureHostedDockerStarter)
     factory.provide("azure_vm", AzureVMExprStarter)
+    factory.provide("k8s_vm", K8sVMExprStarter)
```

Comment:
Suggest using `K8SExprStarter`. Usually uses capitalized `K8S` than `K8s`. And `VM` is actually transparent to K8S users.